testFixtures/database/data/serviceMessages.js

```diff
@@ -1,55 +1,103 @@
 const serviceMessages = [
   {
     message: "Message 1",
-    created_at: new Date("2021-09-20T10:12:13")
+    created_at: () => {
+      const x = new Date()
+      x.setDate(x.getDate() - 270)
+      return x
+    }
```

Comment:
This bit of code is repeated quite a bit - could we pull it out into a reusable function? We could then also use the naming to make it a bit more clear what this is doing (it's not immediately obvious that the number is days!)

Reply:
yup makes sense, will redo